scripts/ci/ansible/deploy.yml

```diff
@@ -145,32 +84,12 @@
       dest: /etc/sudoers.d/jenkins
       owner: root
       group: root
-  - selinux:
-      policy: targeted
-      state: permissive
-  - service:
-      name: "{{ item }}"
-      state: started
-      enabled: yes
-    with_items:
-      - etcd
-      - openvswitch
-      - docker
-      - lxd
-  - shell: "lxd init --auto"
   - lineinfile:
       dest: /etc/hosts
       line: "{{ hostvars['jenkins_master']['private_ip'] }} {{ jenkins_public_hostname }}"
       state: present
-  - shell: |
-      cd /tmp
-      git clone https://github.com/the-tcpdump-group/libpcap.git
-      cd libpcap
-      git checkout libpcap-1.5.3
-      ./configure --prefix=/usr/local --disable-shared --disable-dbus --disable-bluetooth --disable-canusb
-      make
-      sudo make install
-      cd ..
+  roles:
+    - role: skydive_dev
```

Comment:
role skydive_dev is in contrib/ansible/roles
how it would be included ?

Reply:
We can use ANSIBLE_ROLES_PATH variable. Or we can simply create a symlink

Reply:
could you add/set it in  scripts/ci/ansible/run-playbook.sh